### PR TITLE
Add gitdb2

### PR DESCRIPTION
**Pull request by ngoldbaum**
This is needed to update `gitpython` to the latest version.

### CODE REVIEW

File: recipes/gitdb2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - smmap2
+    - setuptools
```
Comment (ocefpaf) [inline recipes/gitdb2/meta.yaml]: Probably don't need `setuptools` at run time.

File: recipes/gitdb2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+about:
+  home: https://github.com/gitpython-developers/gitdb
+  license: BSD 3-Clause
+  license_file: LICENSE
+  summary: 'A pure-Python git object database'
+
+extra:
+  recipe-maintainers:
+    - ngoldbaum
+    - pelson
```
Comment (ocefpaf) [inline recipes/gitdb2/meta.yaml]: Not sure @pelson wants in on this one. We can always add him later though.

**Comment by ngoldbaum** [comment]
Thanks for the review!

File: recipes/gitdb2/meta.yaml (hunk 2 above)
Comment (pelson) [inline recipes/gitdb2/meta.yaml]: 😉

Comment (ocefpaf) [inline recipes/gitdb2/meta.yaml]: Is that a yes or no :smile:

Comment (pelson) [inline recipes/gitdb2/meta.yaml]: It doesn't really make much difference when you have admin rights on conda-forge, right... 😉 

I can't claim to be able to be super responsive on maintenance front 😜 

Basically, if it is there, that's fine. If not, happy to keep it that way.

File: recipes/gitdb2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{% set version = "2.0.2" %}
+{% set sha256 = "f2e36d7561e91f30a6a44858756dc020d8f1e81ca6e4185979d5c6c24c648070" %}
+
+package:
+  name: gitdb
```
Comment (jakirkham) [inline recipes/gitdb2/meta.yaml]: Should this have been called `gitdb2`? Given it overlapped with an existing feedstock, `staged-recipes` didn't overwrite the feedstock. Maybe it should go in as a branch if `gitdb` is the name needed. Alternatively this recipe may need to be readded with a name change. Thoughts @ngoldbaum?

Comment (ngoldbaum) [inline recipes/gitdb2/meta.yaml]: It should have been called `gitdb2`. Can you make a pull request?

Comment (jakirkham) [inline recipes/gitdb2/meta.yaml]: Please see PR ( https://github.com/conda-forge/staged-recipes/pull/3455 ).

**Pull request merged**